### DISCUSSION TITLE
docs/3.x/api-reference/checkout/mutations/checkout-shipping-address-update

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#963] [General]
# docs/3.x/api-reference/checkout/mutations/checkout-shipping-address-update

Update shipping address in the existing checkout.

https://docs.saleor.io/docs/3.x/api-reference/checkout/mutations/checkout-shipping-address-update

**Comment by selbieh** [comment]
i get this error when i try to mutate the shippingAddress 
"errors": [
                {
                    "field": "shippingAddress",
                    "message": "This checkout doesn't need shipping",
                    "code": "SHIPPING_NOT_REQUIRED"
                }
